### DISCUSSION TITLE
unocss is able to use @apply ?

### DISCUSSION THREAD

**Discussion by aliuq** [unocss/unocss#237] [Q&A]
unocss is able to use @apply ?

**Comment by wheatjs** [comment]
Nope, that is going to be left up to other frameworks like windicss. You can read about that here on the article about unocss https://antfu.me/posts/reimagine-atomic-css#what-about-windi-css

**Comment by pierredewilde** [comment]
Yes, it works since v0.26.2.
See https://github.com/unocss/unocss/tree/main/packages/transformer-directives

However, there is still an issue with variants (eg hover:)
See https://github.com/unocss/unocss/issues/649

**Reply by aliuq** [reply]
Thank you
